students/synowiec_krzysztof/lesson_05_lists/numbers_filter.py

```diff
@@ -0,0 +1,9 @@
+def main():
+    strings = [x for x in input().split()]
+    filter = range(int(input("Provide filter\n")))
+    ints = [int(strings[x]) for x in range(len(strings)) if x not in filter]
```

Comment:
Seems that this code filters list based on their indexes. That was not the intention of the task. 
> filters numbers within supplied range

I mean that filtering should be based on actual values, not their position in a list.
Check out updated example:
```
list_of_strings = ['2', '0', '8', '3']	
to_filter_range = range(3)	
expected_output = [8, 3]
```